Lib/test/test_enum.py

```diff
@@ -2893,6 +2893,14 @@ def __new__(cls, c):
         self.assertEqual(FlagFromChar.a, 158456325028528675187087900672)
         self.assertEqual(FlagFromChar.a|1, 158456325028528675187087900673)
 
+    def test_multiple_superclasses(self):
+        class _EnumSuperClass(metaclass=EnumMeta):
+            pass
+        class E(_EnumSuperClass, Enum):
+            A=1
+        self.assertEqual(format(E.A), "E.A")
```

Comment:
I think the test should call repr(), not format()
```suggestion
        self.assertEqual(repr(E.A), "<E.A: 1>")
```

Reply:
Thanks! It was my mistake.